tools/ark-cli/src/commands/export/index.ts

```diff
@@ -0,0 +1,136 @@
+import {Command} from 'commander';
+import {execa} from 'execa';
+import * as fs from 'fs/promises';
+import yaml from 'yaml';
+import type {ArkConfig} from '../../lib/config.js';
+import output from '../../lib/output.js';
+
+const ARK_RESOURCE_TYPES = [
+  'secrets',
+  'tools',
+  'models',
+  'agents',
+  'teams',
+  'evaluators',
+  'mcpservers',
+  'a2aservers',
+];
+
+type ResourceType = (typeof ARK_RESOURCE_TYPES)[number];
+
+interface ExportOptions {
+  output?: string;
+  namespace?: string;
+  types?: string;
+  labels?: string;
+}
+
+async function getResources(
```

Comment:
can we look at the other kubectl commands we have put them into a separate folder, we can do the same here